manifests/kustomize/env/namespaced-installation/kustomization.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+- ../platform-agnostic/minio
+- ../platform-agnostic/mysql
+- ../../base/argo
+- ../../base/pipeline
+- ../../base/metadata
+
+resources:
+- namespace.yaml
+
+namespace: kubeflow
+
+images:
+- name: mysql
+  newTag: "5.6"
+- name: minio/minio
+  newTag: RELEASE.2018-02-09T22-40-05Z
+
+configMapGenerator:
+- name: pipeline-install-config
+  env: params.env
```

Comment:
I tried to use relative path and get this error. Not sure if there's a way to avoid duplication 

```
NewResMapFromSecretArgs: NewResMapFromSecretArgs: env source file: ../../base/params-db-secret.env: security; file '../../base/params-db-secret.env' is not in or below '/Users/jiaxin/go/src/github.com/kubeflow/pipelines/manifests/kustomize/env/namespaced-installation'
```